pipeline/notifications/render_check_message.py

```diff
@@ -0,0 +1,28 @@
+from pipeline.notifications.renderer import RenderNotification
+from pipeline.types import NotificationInfo
+
+
+class NewMessageRender(RenderNotification):
+    def __init__(self):
+        pass
+
+    def render(self, inp: NotificationInfo) -> str:
```

Comment:
I would call this argument `info` or `data` instead of `inp`, to avoid confusion with `NotificationInfo`.